index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB" crossorigin="anonymous">
+
+
+<title>
+</title>
+</head>
+<body>
+<div class="container">
+	<div class="row">
+		<div class='col-xs-12 col-md-4 col-sm-2 gray' id='a1'>1</div>
+        <div class='col-xs-12 col-md-4 col-sm-2 gray-l' id='a2'>2</div>
+        <div class='col-xs-12 col-md-4 col-sm-2 gray' id='a3'>3</div>
+        <div class='col-xs-12 col-md-4 col-sm-2 gray-l' id='b1'>4</div>
+        <div class='col-xs-12 col-md-4 col-sm-2 gray' id='b2'>5</div>
+        <div class='col-xs-12 col-md-4 col-sm-2 gray-l' id='b3'>6</div>
+        <div class='col-xs-12 col-md-4 col-sm-2 gray' id='c1' >7</div>
+        <div class='col-xs-12 col-md-4 col-sm-2 gray-l' id='c2'>8</div>
+        <div class='col-xs-12 col-md-4 col-sm-2 gray' id='c3'>9</div>
+  	</div>	
+</div>	
+<script src="script.js"></script>
+<link rel="stylesheet" type="text/css" href="style.css"> 
+</body>
+</html>
```

Comment:
Its great, I'm glad you know how to use bootstrap to make a responsive website.

But for this case its a bit pointless to make the game board responsive.
You can't play tic tac toe in a single column right ?